readme.md

```diff
@@ -0,0 +1,105 @@
+# Intro
+
+This is MVP for one of our client (UTSEUS) which helps teams on hackathons exchange skills among team members and pay for their help.  It has three modules: web client (React), server (NodeJS) and storage with some business logic on blockchain (ethereum). This doc descriebs how to setup environment and run server backed by blockchian. 
+
+## Install environment
+
+A whole environment is described in scripts of package.json. Almost all what you need is run this scripts from the server folder. 
+
+```
+npm run install
+npm run testrpc
+npm run truffle
+# run server API tests
+npm run test-mocha
+# run unit tests
+npm run test-jest
+```
+
+## Configure truffle.js
+
+Use one of accounts from output of testrpc to put into this truffle.js configuration 
+
+```
+require('babel-register')
+require('babel-polyfill')
+
+module.exports = {
+  networks: {
+    development: {
+      host: "localhost", 
+      port: 8545,
+      network_id: "*",
+      from: "0x041599451d9bac5bee252a5db569856085666f62",
+      gas: 4712388
+    }
+  }
+}
+```
+## Setup users
+Both skills and users are preconfigured on the server. To setup users go to `utils.js` and put new data into array `users` 
+
+## Setup skills
+To setup users go to file `dataset.js` and fill data into variable `skills`
+
+## Start the server
+Move back to the server folder and execute this command
+
+```
+npm run start
+```
+
+## API
+
+/users - to get list of users:
+```
+curl -X GET http://localhost:3000/api/v1/users
+
+{"users":[{"user":"lawlietyumiao"}]}
+```
+
+/skills - to get list of skills:
+
+```
+curl -X GET http://localhost:3000/api/v1/skills
+
+[{"skill":"business model"},{"skill":"financial planning"}]
+```
+
+/help/:mentee/:skill/:time - to register user with skill
+
+```
+curl -X POST http://localhost:3000/api/v1/register/Ricky/React
+
+user was successfully registered
+```
+
+/help/:mentee/:skill/:time - To send request for help where last two parameters are skill and cost: 
+
+```
+curl -X  POST http://localhost:3000/api/v1/help/Dmitry/React/10
```

Comment:
No validation of skills that are actually available?

```
rngadam@rngadam-mac ~/c/s/s/server> curl -X  POST http://localhost:3000/api/v1/help/Dmitry/SomethingThatDoesNotExist/10
{"success":true,"payload":"0x1ea9702e4d1008af77c63b32b228da14ef4eea060ff13411c097be9a3389e18f"}⏎   
```

Reply:
Correct. See above why